threedi_modelchecker/config.py

```diff
@@ -2660,22 +2510,12 @@ def is_none_or_empty(col):
     AllEqualCheck(
         error_code=1152,
         level=CheckLevel.WARNING,
-        column=models.AggregationSettings.timestep,
+        column=models.AggregationSettings.interval,
     ),
-    QueryCheck(
+    AggregationSettingsInvervalCheck(
+        column=models.AggregationSettings.interval,
```

Comment:
Replaced `QueryCheck` with a custom check because `interval` and `output_time_step` are no longer in the same table and cannot be joined (easily) without foreign key.

Reply:
You can, it is called a cartesian join, see comment in the custom check.

`.join(models.GlobalSetting)` would work, but gives a warning.
`.join(models.GlobalSetting, true())` suppresses the warning.